evap/contributor/templates/contributor_index.html

```diff
@@ -152,18 +152,30 @@
                                         {% if evaluation.state != evaluation.State.PUBLISHED and not evaluation.is_single_result %}
                                             {% if evaluation|is_user_editor_or_delegate:user %}
                                                 {% if evaluation.state == evaluation.State.PREPARED %}
-                                                    <a href="{% url 'contributor:evaluation_edit' evaluation.id %}" class="btn btn-primary btn-row-hover"
-                                                        data-bs-toggle="tooltip" data-bs-placement="top" title="{% translate 'Edit or approve' %}">
+                                                    <a href="{% url 'contributor:evaluation_edit' evaluation.id %}" class="btn btn-primary btn-row-hover" data-bs-toggle="tooltip" data-bs-placement="top" title="{% translate 'Edit or approve' %}">
                                                         <span class="fas fa-pencil"></span>
                                                     </a>
                                                     {% if not evaluation|has_nonresponsible_editor %}
-                                                        <button class="btn btn-sm btn-dark" data-bs-toggle="tooltip"
-                                                            data-bs-placement="top" title="{% translate 'Delegate preparation' %}"
-                                                            data-evaluation-name="{{ evaluation.full_name }}"
-                                                            data-delegation-url="{% url 'contributor:evaluation_direct_delegation' evaluation.id %}"
-                                                        >
-                                                            <span class="fas fa-hand-point-left"></span>
-                                                        </button>
+                                                        <form class="d-inline" method="POST" action="{% url 'contributor:evaluation_direct_delegation' evaluation.id %}">
+                                                            {% csrf_token %}
+
+                                                            <confirmation-modal type="submit">
+                                                                <span slot="title">{% translate 'Delegate preparation' %}</span>
+                                                                <span slot="action-text">{% translate 'Delegate preparation' %}</span>
+                                                                <span slot="question">
+                                                                    {% blocktranslate trimmed %}Do you really want to delegate the preparation of the evaluation <strong data-label=""></strong>?{% endblocktranslate %}
```

Comment:
This does not work, data-label was populated by the old modal system but is not supported by confirmation-modal. Instead, the appropriate content should be inserted here directly.

This is possible because now there is a modal for every evaluation whereas before it was shared